### ISSUE TITLE
UX: Color coding the transfers with different status on the 'Downloads' tab

### ISSUE THREAD

**Issue by 501NotImplemented** [nicotine-plus/nicotine-plus#2940]
### Describe the solution you'd like
As a user, I would like to have the ability to highlight the transfers of **different** statuses by **different** colors (_just like in the original Soulseek client_), to visually separate them in the grid and have a better user experience.

As of now, the only way to anchor the attention is the transfer progress bar, however the color of transfer in unfinished status (<100%) and completed (100%) is the same, blue one.

### Additional context
Basically, after migrating to **Nicotine+** from **Soulseek QT**, I miss the color coding of different downloads on the downloads tab. It would be much easier to visually separate them on the downloads tab, the SoulseekQT color theme will work perfectly for me.
E.g. 
- Finished - green
- Connection timeout, canceled - red
- User logged off - grey out
- Queued - light blue
- Transferring - black

**Comment by mathiascode** [comment]
Greyed out "User logged off" rows have been added (reuses the same mechanism as greyed out search results).

I don't want to add colored rows until we've ported treeviews to Gtk.ColumnView, because text styling is done in a completely different way there.
